library/aes.c

```diff
@@ -1031,13 +1058,18 @@ int mbedtls_aes_crypt_ctr( mbedtls_aes_context *ctx,
                        const unsigned char *input,
                        unsigned char *output )
 {
-    int c, i;
+    int c, i, ret = 0;
     size_t n = *nc_off;
+    unsigned char *out = output;
 
```

Comment:
Is there a missing check here?
```
if( n > 0x0F )
```

Reply:
I don't think so. `mbedtls-2.7` doesn't contain this check

Reply:
Ok, but should it be there?

Reply:
Although I agree it's a good enhancement as parameter validation, it's not part of the scope of the PR